### PR TITLE
i18n(zh-cn): update error reference docs

### PR DESCRIPTION
**Pull request by 100gle**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

update error reference docs since #4785

#### Related Issue / Implementation PR

- Closes Issue #

### CODE REVIEW

File: src/content/docs/zh-cn/reference/errors/markdown-image-not-found.mdx [hunk 1]
```diff
@@ -4,6 +4,10 @@ i18nReady: true
 githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
 ---
 
+:::caution[废弃的]
+该错误不再是 Markdown 所特有，已被 `ImageNotFound` 错误替代。
+:::
```
Comment (Genteure) [inline src/content/docs/zh-cn/reference/errors/markdown-image-not-found.mdx:9]:
```suggestion
:::caution[已废弃]
该错误不再是 Markdown 特有的，因此已被 `ImageNotFound` 替代。
:::
```

Rewording to be more natural.

**Review by Genteure — COMMENTED** [review]
Small suggestion

**Review by Genteure — APPROVED** [review]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Great work peeps 🙌

**Pull request merged**